### DISCUSSION TITLE
Is it possible to save a linked_map as html?

### DISCUSSION THREAD

**Discussion by yohman** [opengeos/leafmap#631] [Q&A] [answered]
If not, would love to see this feature enabled. Loving leafmap, amazing work team!

**Comment by giswqs** [comment, marked as answer]
Unfortunately, the answer is No. The linked map requires a Jupyter kernel, which HTML does not have. The alternatives are voila and solsar.

- https://github.com/opengeos/voila-geospatial
- https://github.com/opengeos/solara-geospatial

**Reply by yohman** [reply]
Thanks, I thought as much. Qiusheng, I love your work. We should get together sometime.
